website/content/en/docs/installation/_index.md

```diff
@@ -80,7 +80,7 @@ operator-sdk_linux_amd64: OK
 #### 3. Install the release binary in your PATH
 
 ```sh
-chmod +x operator-sdk_${OS}_${ARCH} && sudo mv operator-sdk_${OS}_${ARCH} /usr/local/bin/operator-sdk
+sudo chmod +x operator-sdk_${OS}_${ARCH} && sudo mv operator-sdk_${OS}_${ARCH} /usr/local/bin/operator-sdk
```

Comment:
Why do we need the `sudo` for `chmod`? The idea is that it would've been downloaded as yourself so you'd have permissions to change the mode. The `sudo` is only required to copy it to a location in your path that you probably would not have access to.

Reply:
I required `sudo` for some reason even though I wasn't running as elevated user.